src/TimeStruct.jl

```diff
@@ -47,8 +47,10 @@ export repr_periods
 export strat_periods, strategic_periods
 export regular_tree, strat_nodes, strategic_scenarios
 export withprev, chunk, chunk_duration
-export isfirst, duration, duration_strat, multiple, probability
+export isfirst, duration, duration_strat, multiple
+export probability, probability_scen
```

Comment:
Shall we also export `probablity_branch`? It could make sense now.

Reply:
Done